src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -46,6 +46,8 @@
 import com.cloudbees.jenkins.plugins.bitbucket.server.client.repository.BitbucketServerRepository;
 import com.cloudbees.jenkins.plugins.bitbucket.server.client.repository.BitbucketServerWebhooks;
 import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import com.google.common.base.Supplier;
```

Comment:
hrm guava use - that has been historically a source of breakages when the (not very well controlled or specified) verion of guava is bumped...

Reply:
I don't think there is any reason why we cant depend on Java 8 for this component now - WDYT @amuniz ?